src/sql/src/error.rs

```diff
@@ -145,6 +145,14 @@ pub enum Error {
         statement: String,
         location: Location,
     },
+    #[snafu(display("Unable to convert sql value {} to datatype {:?}", value, datatype))]
```

Comment:
nitpick: add a new line
```suggestion

    #[snafu(display("Unable to convert sql value {} to datatype {:?}", value, datatype))]
```